yt_dlp/extractor/mxplayer.py

```diff
@@ -106,73 +106,93 @@ class MxplayerIE(InfoExtractor):
+            'description': 'Jatin and Karishma, who meet each other on a plane to India, are set to marry the partners chosen by their respective parents. However, they find themselves attracted to each other.',
+        },
+        'params': {
+            'format': 'bv',
+            'skip_download': True,
+        },
+    }, {
+        'url': 'https://www.mxplayer.in/show/watch-ek-thi-begum-hindi/season-2/game-of-power-online-5e5305c28f1409847cdc4520b6ad77cf',
+        'info_dict': {
+            'id': '5e5305c28f1409847cdc4520b6ad77cf',
+            'display_id': 'game-of-power-online',
+            'title': 'Game Of Power',
+            'duration': 1845,
+            'ext': 'mp4',
+            'description': 'Cops arrest Ashraf assuming a prostitute. But she gets away with her new identity, Leela Paswan. Maqsood dumps Nana Mhatre and appoints Shaqeel Ansari as a new Bombay chief. For society, he is a sophisticated businessman. The rift between the new chief minister, Yashwant Patil and Shaqeel Ansari begins with a failed business deal. Ashwin Surve, a daredevil young gangster challenges established gangs of Maqsood & Bhai Chavan. He wants to enter in the drug business. But the only supplier Nari Khan supplies stuff only to Maqsood gang and no one else. ACP Qureshi arrests Bhai Chavan and turns his focus on people close to Zaheer and Ashraf. Ashraf faces a narrow escape from Qureshi and decides to take up her new identity beyond just fake documents.',
+            'series': 'Ek Thi Begum (Hindi)',
+            'season': 'Season 2',
+            'season_number': 2,
+            'episode': 'Episode 2',
+            'episode_number': 2,
+        },
+        'params': {
+            'format': 'bv',
+            'skip_download': True,
+        },
     }]
 
     def _real_extract(self, url):
-        type, display_id, video_id = self._match_valid_url(url).groups()
-        type = 'movie_film' if type == 'movie' else 'tvshow_episode'
-        API_URL = 'https://androidapi.mxplay.com/v1/detail/'
-        headers = {
-            'X-Av-Code': '23',
-            'X-Country': 'IN',
-            'X-Platform': 'android',
-            'X-App-Version': '1370001318',
-            'X-Resolution': '3840x2160',
-        }
-        data_json = self._download_json(f'{API_URL}{type}/{video_id}', display_id, headers=headers)['profile']
-
-        season, series = None, None
-        for dct in data_json.get('levelInfos', []):
-            if dct.get('type') == 'tvshow_season':
-                season = dct.get('name')
-            elif dct.get('type') == 'tvshow_show':
-                series = dct.get('name')
-        thumbnails = []
-        for thumb in data_json.get('poster', []):
-            thumbnails.append({
-                'url': thumb.get('url'),
-                'width': thumb.get('width'),
-                'height': thumb.get('height'),
-            })
+        video_type, display_id, video_id = self._match_valid_url(url).groups()
```

Comment:
use `group` instead `groups`
```suggestion
        video_type, display_id, video_id = self._match_valid_url(url).group('type', 'display_id', 'id')
```